Overview.src.html

```diff
@@ -3216,6 +3266,59 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <li>If <var>cookies</var> is not the empty string, append
      `<code title=http-cookies>Cookie</code>`/<var>cookies</var> to <var>httpRequest</var>'s
      <span title=concept-request-header-list>header list</span>.
+
+     <li><p>If the User Agent supports Token Binding,
+       build an appropriate <code title="">Sec-Token-Binding</code> header by running these substeps:
```

Comment:
I don't think we need to say "build an appropriate ..." here as that follows from the steps and the new phrasing might put people off.